### PR TITLE
Minor text improvements

### PR DESCRIPTION
**Pull request by sypets**

### CODE REVIEW

File: Documentation/Index.rst [hunk 1]
```diff
@@ -48,12 +48,11 @@ at hand.
 
 **Versions**
 
-TCA properties change between major versions, this main array had significant
+TCA properties change between major versions. This manual had significant
 refactorings especially between core versions 6.2 to 7 LTS, and again between 7
```
Comment (mbrodala) [inline Documentation/Index.rst:52]: Since this part is 1) outdated and 2) true for any LTS we should rephrase it to generally mention possible changes between LTS versions.

Comment (linawolf) [reply Documentation/Index.rst:52]: do you have an idea, how to rephrase it? I was also thinking about the sentence but had no idea.

**Review by linawolf — COMMENTED** [review]
I would also

File: Documentation/Index.rst (hunk 1 above)
Comment (mbrodala) [reply Documentation/Index.rst:52]: _... refactorings between each TYPO3 LTS release._

This should be enough.

Comment (mbrodala) [reply Documentation/Index.rst:52]: Also "refactoring" is probably the wrong term since this is mostly meant for actual code. So we should maybe use "changes" or similar instead.

**Review by linawolf — APPROVED** [review]
